src/config/WarehouseCrawlDataConfig.ts

```diff
@@ -0,0 +1,41 @@
+const URL = 'https://www.cbre-propertysearch.jp/industrial/';
+
+const LIST_PROVINCE = {
+  DOM_PROVINCE : '#contents > div > div.propertyList > div > div.itemGroup >.item',
+  DOM_WAREHOUSE : 'div.inner > div > div.body > div.head > h2 > a',
+  DOM_LOCATION : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(1) > td',
+  DOM_TRAFFIC : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(2) > td',
+  DOM_SCALE : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(3) > td',
+  DOM_COMPLETION : 'div.inner > div > div.body > div.info > div > table > tbody > tr:nth-child(4) > td',
+};
+
+const LIST_STORE = {
+  DOM_STORE: '#contents > div > div.columnSection.clearfix',
+  DOM_IMAGE_1: 'div > div.imgSection > div > div.photo > div.inner > div > div:nth-child(1) > img',
+  DOM_IMAGE_2: 'div > div.imgSection > div > div.photo > div.inner > div > div:nth-child(2) > img',
+  DOM_NAME: 'div > div.bodySection > table > tbody > tr:nth-child(1) > td',
+  DOM_LOCATION: 'div > div.bodySection > table > tbody > tr:nth-child(2) > td',
+  DOM_TRAFFIC: 'div > div.bodySection > table > tbody > tr:nth-child(3) > td',
+  DOM_DISTRICT: 'div > div.bodySection > table > tbody > tr:nth-child(4) > td',
+  DOM_COVERAGE_RATIO: 'div > div.bodySection > table > tbody > tr:nth-child(5) > td',
+  DOM_DATE: 'div > div.bodySection > table > tbody > tr:nth-child(6) > td',
+  DOM_SCALE: 'div > div.bodySection > table > tbody > tr:nth-child(7) > td',
+  DOM_CONSTRUCTION: 'div > div.bodySection > table > tbody > tr:nth-child(8) > td',
+  DOM_TOTAL: 'div > div.bodySection > table > tbody > tr:nth-child(9) > td',
+  DOM_ELEVATOR: 'div > div.bodySection > table > tbody > tr:nth-child(10) > td',
+  DOM_CEILING_HEIGHT: 'div > div.bodySection > table > tbody > tr:nth-child(11) > td',
+  DOM_FLOOR_LOAD: 'div > div.bodySection > table > tbody > tr:nth-child(12) > td',
+  DOM_FLOOR: 'div > div.bodySection > table > tbody > tr:nth-child(13) > td',
+  DOM_FLOOR_TYPES: 'div > div.bodySection > table > tbody > tr:nth-child(14) > td',
+  DOM_EQUIPMENT: 'div > div.bodySection > table > tbody > tr:nth-child(15) > td',
+  DOM_BUILDING: 'div > div.bodySection > table > tbody > tr:nth-child(16) > td',
```

Comment:
xử lí quá dài
Chỉ cần lưu cái DOM đến thằng tbody `div > div.bodySection > table > tbody`
xong r dùng vòng for để lấy toàn bộ content trong cái bảng đó là được

Reply:
đã xử lý